basie/user_templates/test_targets.txt

```diff
@@ -0,0 +1,10 @@
+Alpha   EqCross1_3  TP EQ  12:00:00h +45:00:00
```

Comment:
I added this file reproducing the test cases that were needed for some tests but were removed in some recent version of the example files.